config/custom.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Custom Configuration
+ *
+ * https://craftcms.com/docs/4.x/config/#custom-settings
+ */
+
+return [
+    // Global settings
+    '*' => [
+        'enableGoogleAnalytics' => false,
+    ],
+
+    // Dev environment
+    'dev' => ['enableGoogleAnalytics' => true],
```

Comment:
Why is this enabled on dev? We only want the code to be active on production, right?

Reply:
I thought, initially it would be helpful to keep analytics enabled for development and testing. But the ideal case would be to keep it enabled for production only.

I have now disabled the analytics for the development environment.

Reply:
Developers can always tweak this value when they want to test. But the default config should reflect the shippable state not the testing state.